src/pages/homePage/HomePage.css

```diff
@@ -0,0 +1,35 @@
+.homepage {
+    width: 100%;
+    height: 100%;
+    top: 0px;
+    position: absolute;
+    width: 100%;
+    margin-right: 0px;
+    margin-left: 0px;
+    background-image: url(./default.png);
+    filter: blur(8px);
+    background-position: left;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.home_description {
+    margin-right: auto;
+    margin-left: auto;
+    position: relative;
+    top: 400px;
+    color: white;
+    background-color: #789778;
+    border-radius: 10px;
+    padding: 100px;
+    box-shadow: rgba(0, 0, 0, 0.2) 0px 0px 0px 1px;
```

Comment:
If I remember correctly, when you write`0` you don't need to add the unit `px` afterwards.

Reply:
True, I removed the unit `px`.